source/extended-json.rst

```diff
@@ -338,6 +338,48 @@ to parsing JSON numbers:
   this results in a loss of precision.  The parser MUST NOT interpret it as a
   BSON String containing a decimal representation of the number.
 
+Special rules for parsing ``$uuid`` fields
+..........................................
+
+As per the `UUID specification`_, Binary subtypes 3 and 4 are used to
+represent UUIDs in BSON. Normally, UUIDs are represented in extended JSON
+as defined in the `Conversion table`_, e.g. the following document written
+in Groovy with the MongoDB Java Driver::
+
+  {"Binary": UUID.fromString("c8edabc3-f738-4ca3-b68d-ab92a91478a3")}
+
+is transformed into the following (newlines and spaces added for readability)::
+
+  {"Binary": {
+      "$binary": {
+          "base64": "o0w498Or7cijeBSpkquNtg==",
+          "subType": "03"}
+      }
+  }
+
+While this transformation preserves BSON subtype information (since
+UUIDs can be represented as BSON subtype 3 *and* 4), it is not very
+human readable. Since extended JSON representations of UUIDs feature
```

Comment:
Is `not very human readable` accurate?  It seems more relevant (and sticking more to fact than opinion) to say that base64 is not the standard way of representing UUIDs, so it's harder to compare against the string representation coming from platform libraries.

Reply:
Done.